x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/191_lookup_join_text.yml

```diff
@@ -0,0 +1,133 @@
+---
+setup:
+  - requires:
+      test_runner_features: [capabilities, contains]
+      capabilities:
+        - method: POST
+          path: /_query
+          parameters: []
+          capabilities: [lookup_join_text]
```

Comment:
Uh, I missed it, but this should also declare `join_lookup_v11`. 
This should help skipping this spec if something changes about basic lookup logic (such as grammar :) )

Reply:
Opened a pr for it https://github.com/elastic/elasticsearch/pull/120771